aries_cloudagent/resolver/did_resolver.py

```diff
@@ -9,10 +9,15 @@
 from itertools import chain
 from typing import Union
 
-from pydid import DID, DIDUrl, DIDDocument, Service, VerificationMethod
+from pydid import DID, DIDUrl, DIDDocument, Service, VerificationMethod, DIDError
```

Comment:
alphabetical order here helps humans find imports in troubleshooting if need be

Reply:
I agree! I have sorted the relevant imports.